dot/rpc/service.go

```diff
@@ -91,8 +91,8 @@ func (s *Service) BuildMethodNames(rcvr interface{}, name string) {
 
 // isExported returns true of a string is an exported (upper case) name.
 func isExported(name string) bool {
-	rune, _ := utf8.DecodeRuneInString(name)
-	return unicode.IsUpper(rune)
+	r, _ := utf8.DecodeRuneInString(name)
```

Comment:
why do we not check this error?

Reply:
utf8.DecodeRuneInString returns rune and size, the second ignored var is for size not for error.